cancer_ai/validator/model_runners/pytorch_runner.py

```diff
@@ -1,16 +1,17 @@
 from . import BaseRunnerHandler
-from typing import List, AsyncGenerator
+from typing import List, AsyncGenerator, Union, Dict, Any, Tuple
```

Comment:
revert whole file

Reply:
we don't have other runners, please revert. also this logic shouldn't be there, it should be in competition-speficif handler. this is the problem I solved with my original PR and let's no go back